Description: use SIMD_everywhere for compilation on other architectures
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-04-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/scene/import-texture.cpp
+++ b/src/scene/import-texture.cpp
@@ -15,7 +15,8 @@
 using vulkan::setImageName;
 
 #include <FreeImage.h>
-#include <immintrin.h>
+#define SIMDE_ENABLE_NATIVE_ALIASES
+#include <simde/x86/avx.h>
 
 static FIBITMAP *loadBitmap(string filename, VkFormat *format)
 {
